apis/python/src/tiledb/vector_search/object_readers/bioimage_reader.py

```diff
@@ -0,0 +1,210 @@
+from typing import Any, Mapping, Optional, List, Tuple, Dict, OrderedDict
+# from tiledb.vector_search.object_readers import ObjectPartition, ObjectReader
+from tiledb import Attr
+
+
+# class BioImagePartition(ObjectPartition):
```

Comment:
Should this be commented back in? As-is, we're not a child class of `ObjectPartition`, right? Same with other Partitions and Readers.